tools/changelog/cmd/render.go

```diff
@@ -0,0 +1,161 @@
+// Copyright 2016-2022, Pulumi Corporation.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmd
+
+import (
+	"context"
+	"fmt"
+	"io/fs"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strings"
+	"time"
+
+	"github.com/goccy/go-yaml"
+	"github.com/google/go-github/v47/github"
+	"github.com/pulumi/watchutil-rs/tools/changelog/changelog"
+	"github.com/spf13/cobra"
+)
+
+var inDir string
```

Comment:
This stuff ends up accessible to the entire "cmd" module. I much prefer the style we do in pulumi/pulumi where the vars are declared in a method with the cmd, it keeps them scoped to just that command. See https://github.com/pulumi/pulumi/blob/master/pkg/cmd/pulumi/about.go#L45 for an example.

Reply:
Will do to avoid polluting package scope.